MyFirstDb/DataModel/Team.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MyFirstDb.DataModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace MyFirstDb.DataModel
+{
+    public class Team
+    {
+        public int Id { get; set; }
+        [MaxLength(100)][Required]
```

Comment:
Add please empty line
```suggestion

        [MaxLength(100)][Required]
```